packages/platform-sdk-generator/src/model/Model.ts

```diff
@@ -175,7 +175,10 @@ export class Model {
   }
 
   #addComponent(c: ir.Component) {
-    const ns = this.#namespaces.get(c.namespace ?? "");
+    const nsName = (c.namespace === undefined || c.namespace === "Core")
```

Comment:
lets do `==null`, read https://pages.github.palantir.build/palantir/documentation/#/gotham/web-standards/typescript/explicitly-check-null

Reply:
Will add in the legacy client long aggregation fix PR